cloud/defaults.go

```diff
@@ -176,6 +187,12 @@ func NATRuleID(subscriptionID, resourceGroup, loadBalancerName, natRuleName stri
 	return fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/loadBalancers/%s/inboundNatRules/%s", subscriptionID, resourceGroup, loadBalancerName, natRuleName)
 }
 
+// AvailabilitySetID returns the azure resource ID for a given availability set.
+func AvailabilitySetID(subscriptionID, resourceGroup, availabilitySetName string) string {
+	///subscriptions/eba22861-4990-4213-ab8e-776393e3696b/resourceGroups/cl-as-ncus/providers/Microsoft.Compute/availabilitySets/cl-as-ncus_control-plane-as
```

Comment:
is this comment intentional or did you forget to clean it up?

Reply:
Fixed it, Thanks!